README.MD

```diff
@@ -1,41 +1,52 @@
 
-<!--
-**Note** that for issues related to ***versions of Business Central that are in mainstream support you should use the support channel*** - see [this FAQ entry](https://github.com/Microsoft/AL/wiki/Frequently-Asked-Questions#i-found-an-issue-in-a-version-of-the-product-that-is-not-the-latest-what-do-i-do). -->
+The main purpose of this repository is to track issues regarding the latest version of the AL compiler and developer tools available in the [Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=ms-dynamics-smb.al) or as part of the [AL Developer Preview builds](#al-developer-preview-builds) for Dynamics 365 Business Central. In addition, you will find code samples to help you get started writing extensions in AL and get familiar with the tools.
+
+## Documentation
+
+For documentation about using the [AL Language](https://marketplace.visualstudio.com/items?itemName=ms-dynamics-smb.al) to extend Dynamics 365 Business Central, read some of our topics on [Microsoft Dynamics 365 Business Central docs](https://docs.microsoft.com/en-us/dynamics365/business-central/):  
+• [Getting Started](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-get-started)  
+• [Building Your First Sample Extension](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-extension-example)  
+• [Developer Reference](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-reference-overview)  
+
+## Bugs and Feature Requests
+
+If you have encountered a bug or you have a feature request, please first read the [contributing guidelines](CONTRIBUTING.md) and then search for existing and closed issues.  
+
+Bugs pertaining to the AL compiler and the developer tools for the latest version can be filed in this repository. We will respond to these issues on a best-effort basis.  
+
+See the [triage process](https://github.com/Microsoft/AL/wiki/Triage-Process) document for more information on how we handle incoming requests.
 
-**Note** that for issues related to: 
+## Community
 
-- **versions of Business Central that are in mainstream support you should use the support channel** - see [this FAQ entry](https://github.com/Microsoft/AL/wiki/requently-Asked-Questions#i-found-an-issue-in-a-version-of-the-product-that-is-not-the-latest-what-do-i-do).
+Our answers to frequently asked questions can be found in our [FAQ](https://github.com/Microsoft/AL/wiki/Frequently-Asked-Questions). To get help from community members please use one of the following resources:
 
-- For issues related to **bugs on AL Language VSIX** in Visual Studio Code (aka ModernDev), use the current [GitHub repo](https://github.com/microsoft/al).
+- [Business Central Community](https://community.dynamics.com/business/f/758)  
+- [mibuso forum](https://forum.mibuso.com/categories/nav-three-tier)  
+- [Dynamics User Group](https://dynamicsuser.net/nav/f/developers)  
```

Comment:
(not mentioning Office Hours here as not supporting cross partner collaboration)
```suggestion
- [Dynamics User Group](https://dynamicsuser.net/nav/f/developers)  
- [Yammer groups for partners] (https://aka.ms/BCYammer)
```